src/User/Console/CreateCommand.php

```diff
@@ -2,9 +2,9 @@
 
 declare(strict_types=1);
 
-namespace App\Command\User;
+namespace App\User\Console;
```

Comment:
Why commands inside a `Console` namespace, but controllers does not have own `Controller`?
Maybe it worse to add additional level of context: `Console`, `Controller`, `Entity`, `Repository` etc

Reply:
Yes, makes sense when there are multiple objects of the same type.

Reply:
Same for https://github.com/yiisoft/yii-demo/tree/group-by-usecase/src/Blog